libr/search/search.c

```diff
@@ -42,6 +50,7 @@ R_API RSearch *r_search_new(int mode) {
 R_API RSearch *r_search_free(RSearch *s) {
 	if (!s) return NULL;
```

Comment:
IMHO the following two alternatives enhance readability. Also see my comment above.
```c
if (!s) return NULL;
```

```c
if (!s)
  return NULL;
```